### DISCUSSION TITLE
The status of pool is always "Ready"?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#10941] [Q&A]
Pool had beed deleted by "ceph osd pool rm", but the status of the pool is always "Ready"?

```
❯ kubectl get cephblockpool -n rook-ceph
NAME           PHASE
replicapool    Ready
**replicapool1   Ready**
```

```
❯ rados lspools
device_health_metrics
zhucan2
zhucan3
zhucan4
zhucan5
replicapool
myfs-metadata
myfs-replicated
my-store.rgw.buckets.index
my-store.rgw.buckets.non-ec
my-store.rgw.meta
my-store.rgw.log
my-store.rgw.control
.rgw.root
my-store.rgw.buckets.data
```

**Comment by travisn** [comment]
The operator does not actively check for the existence of the pools. Only during a pool CR reconcile will the operator check for pools. It's just not expected for a pool to be deleted. To re-create the pool, you can restart the operator. If you want the operator to actively check for pools, please describe the scenario more.

**Reply by zhucan** [reply]
It's needed to monitor the status of the pool, and sync it by operator?

**Reply by travisn** [reply]
Rook's design is currently to watch for K8s events, which will result in reconcile and updated ceph resources. But Rook doesn't usually watch for Ceph events since Rook is only expected to configure Ceph. But PRs are welcome if you want to change this behavior to watch for pool deletion.
